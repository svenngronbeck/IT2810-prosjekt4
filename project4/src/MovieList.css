#movieList{
  margin-left: 20%;
  display: flex;
  flex-wrap: wrap;
  margin-top: 3%;
  width: 60%;
}
