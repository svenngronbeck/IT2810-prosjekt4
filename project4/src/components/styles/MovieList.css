#movieList{
    display: flex;
    margin-top: 2%;
    width: 85%;
    border-radius: 5px;
 }

.innerMovieContainerBig{
    width: 70%;
    margin-left: 12.5%;
}

#movieListInner{
    display: flex;
    justify-content: flex-start;
    flex-wrap: wrap;
    width: 100%
}

@media screen and (max-width: 1500px) {
    .innerMovieContainerBig{
        width: 90%;
        margin-left: 1.5%;
    }
}