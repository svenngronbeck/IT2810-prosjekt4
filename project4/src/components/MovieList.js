import React, { Component } from 'react';
import Movie from './Movie';
//styles
import '../MovieList.css';

class MovieList extends Component {

  //-----RENDER----------
  render() {

    return (
      <div id="movieList">
        <Movie movieID={1} title="'Arry Pottah" showInfo={this.props.showInfo} description="Yer a wizard, 'Arry" imgsrc={"https://imgc.allpostersimages.com/img/print/plakater/harry-potter-philosophers-stone-movie-poster_a-G-9843009-0.jpg"}/>
        <Movie title="John Wick" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://images-na.ssl-images-amazon.com/images/I/51-v2kovwfL._SY445_.jpg"} />
        <Movie title="John Wick 2" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://secure.sfanytime.se/movieimages/COVER/6a9bcb13-2366-4704-9d80-a75200e8f7f8_COVER_01.jpg"}/>
        <Movie title="John Wick" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://images-na.ssl-images-amazon.com/images/I/51-v2kovwfL._SY445_.jpg"}/>
        <Movie title="John Wick" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://images-na.ssl-images-amazon.com/images/I/51-v2kovwfL._SY445_.jpg"}/>
        <Movie title="John Wick" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://images-na.ssl-images-amazon.com/images/I/51-v2kovwfL._SY445_.jpg"}/>
        <Movie title="John Wick" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://images-na.ssl-images-amazon.com/images/I/51-v2kovwfL._SY445_.jpg"} />
        <Movie  title="John Wick" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://images-na.ssl-images-amazon.com/images/I/51-v2kovwfL._SY445_.jpg"}/>
        <Movie title="John Wick" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://images-na.ssl-images-amazon.com/images/I/51-v2kovwfL._SY445_.jpg"}/>
        <Movie title="John Wick" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://images-na.ssl-images-amazon.com/images/I/51-v2kovwfL._SY445_.jpg"}/>
        <Movie movieID={11} title="John Wick" showInfo={this.props.showInfo} description="This is a description" imgsrc={"https://images-na.ssl-images-amazon.com/images/I/51-v2kovwfL._SY445_.jpg"}/>
      </div>
    );
  }
}

export default MovieList;
