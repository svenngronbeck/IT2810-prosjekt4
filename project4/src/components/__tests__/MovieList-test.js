import React from 'react';
import renderer from 'react-test-renderer';
import MovieList from "../MovieList";
import {Provider} from "mobx-react";

describe("MovieList", () => {
    const testStore = {
        movies: [
            {
                _id: "testID",
                title: "test",
                year: 1962,
                rated: "S",
                runtime: 99,
                countries: [],
                genres: [
                    "test"
                ],
                director: "test",
                writers: [
                    "test"
                ],
                actors: [
                    "test",
                ],
                plot: null,
                poster: null,
                imdb: {
                    id: "tt0132936",
                    rating: 5.1,
                    votes: 25
                },
                awards: {
                    wins: 0,
                    nominations: 0,
                    text: ""
                },
                type: "test",
                ratings: [1,2],
            },
            {
                _id: "testID2",
                title: "test2",
                year: 1969,
                rated: "S",
                runtime: 69,
                countries: [],
                genres: [
                    "test2"
                ],
                director: "test2",
                writers: [
                    "test2"
                ],
                actors: [
                    "test2",
                ],
                plot: null,
                poster: null,
                imdb: {
                    id: "tt0132969",
                    rating: 6.9,
                    votes: 69
                },
                awards: {
                    wins: 0,
                    nominations: 0,
                    text: ""
                },
                type: "test",

            },

    ]
    };

    //Simple snapshot testing
     test('MovieList renders correctly?', () => {
         const component =
             (<Provider movieStore = {testStore}>
                 <MovieList movieStore = {testStore}/>
             </Provider>);
         const instance = renderer.create(component).toJSON();
         expect(instance).toMatchSnapshot();
     });
});