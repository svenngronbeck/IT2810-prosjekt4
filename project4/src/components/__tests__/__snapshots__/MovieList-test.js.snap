// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieList MovieList renders correctly? 1`] = `
<div
  id="movieList"
>
  <div
    id="movieListInner"
  >
    <div
      className="react-notification-root"
    >
      <div
        className="notification-container-top-left"
      />
      <div
        className="notification-container-top-right"
      />
      <div
        className="notification-container-bottom-left"
      />
      <div
        className="notification-container-bottom-right"
      />
      <div
        className="notification-container-top-center"
      />
      <div
        className="notification-container-bottom-center"
      />
    </div>
    <div
      className="movieContainerSmall"
      id="mainMovieContainer"
    >
      <div
        className="innerMovieContainerSmall"
        id="testID"
        onClick={[Function]}
      >
        <a
          className="movieLink"
          href="#testID"
        >
          <img
            alt="Img not found.."
            className="movieImageSmall"
            onError={[Function]}
            src="noimage.png"
          />
        </a>
        <div
          className="movieInfoHide"
        >
          <div
            className="bigMovieTitle"
          >
            test
          </div>
          <div
            className="movieDescription"
          />
          <div
            className="movieGenre"
          >
            Genre: 
            test 
          </div>
          <div
            className="movieRatings"
            onClick={[Function]}
          >
            <div
              className="imdbRatings"
            >
              <a
                href="https://www.imdb.com/title/tt0132936"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt=""
                  className="ratingImage"
                  src="imdb.png"
                />
              </a>
              (5.1)
            </div>
            <div
              className="userRatings"
            >
              <p
                className="userRatingText"
              >
                User rating:
              </p>
              <div
                className="star-ratings"
                style={
                  Object {
                    "boxSizing": "border-box",
                    "display": "inline-block",
                    "position": "relative",
                  }
                }
                title="1.5 Stars"
              >
                <svg
                  className="star-grad"
                  style={
                    Object {
                      "height": "0",
                      "position": "absolute",
                      "visibility": "hidden",
                      "width": "0",
                      "zIndex": "0",
                    }
                  }
                >
                  <defs>
                    <linearGradient
                      id="starGrad576209814354507"
                      x1="0%"
                      x2="100%"
                      y1="0%"
                      y2="0%"
                    >
                      <stop
                        className="stop-color-first"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "#eac600",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-first"
                        offset="50%"
                        style={
                          Object {
                            "stopColor": "#eac600",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-final"
                        offset="50%"
                        style={
                          Object {
                            "stopColor": "rgb(203, 211, 227)",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-final"
                        offset="100%"
                        style={
                          Object {
                            "stopColor": "rgb(203, 211, 227)",
                            "stopOpacity": "1",
                          }
                        }
                      />
                    </linearGradient>
                  </defs>
                </svg>
                <div
                  className="star-container"
                  onClick={null}
                  onMouseEnter={null}
                  onMouseLeave={null}
                  style={
                    Object {
                      "cursor": undefined,
                      "display": "inline-block",
                      "paddingLeft": undefined,
                      "paddingRight": undefined,
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "#eac600",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
              </div>
              <p
                className="userRatingText"
              >
                (
                1.50
                ) - 
                2
                 votes
              </p>
            </div>
            <div
              className="giveUserRating"
            >
              <p
                className="userRatingText"
              >
                Give user-rating: 
              </p>
              <div
                className="star-ratings"
                style={
                  Object {
                    "boxSizing": "border-box",
                    "display": "inline-block",
                    "position": "relative",
                  }
                }
                title="0 Stars"
              >
                <svg
                  className="star-grad"
                  style={
                    Object {
                      "height": "0",
                      "position": "absolute",
                      "visibility": "hidden",
                      "width": "0",
                      "zIndex": "0",
                    }
                  }
                >
                  <defs>
                    <linearGradient
                      id="starGrad328891447867881"
                      x1="0%"
                      x2="100%"
                      y1="0%"
                      y2="0%"
                    >
                      <stop
                        className="stop-color-first"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "#eac600",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-first"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "#eac600",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-final"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "rgb(203, 211, 227)",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-final"
                        offset="100%"
                        style={
                          Object {
                            "stopColor": "rgb(203, 211, 227)",
                            "stopOpacity": "1",
                          }
                        }
                      />
                    </linearGradient>
                  </defs>
                </svg>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": undefined,
                      "paddingRight": "2px",
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": "2px",
                      "paddingRight": "2px",
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": "2px",
                      "paddingRight": "2px",
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": "2px",
                      "paddingRight": "2px",
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": "2px",
                      "paddingRight": undefined,
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="smallImgStarShow"
        >
          <a
            href="https://www.imdb.com/title/tt0132936"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt=""
              className="ratingImage"
              src="imdb.png"
            />
          </a>
          <div>
            5.1
             / 10
          </div>
        </div>
        <a
          className="movieLink"
          href="#testID"
        >
          <div
            className="titleTextSmallShow"
          >
            test
          </div>
        </a>
        <div
          className="rating"
        />
        <img
          alt=""
          className="hideExpandImage"
          src="expanded.png"
        />
      </div>
    </div>
    <div
      className="react-notification-root"
    >
      <div
        className="notification-container-top-left"
      />
      <div
        className="notification-container-top-right"
      />
      <div
        className="notification-container-bottom-left"
      />
      <div
        className="notification-container-bottom-right"
      />
      <div
        className="notification-container-top-center"
      />
      <div
        className="notification-container-bottom-center"
      />
    </div>
    <div
      className="movieContainerSmall"
      id="mainMovieContainer"
    >
      <div
        className="innerMovieContainerSmall"
        id="testID2"
        onClick={[Function]}
      >
        <a
          className="movieLink"
          href="#testID2"
        >
          <img
            alt="Img not found.."
            className="movieImageSmall"
            onError={[Function]}
            src="noimage.png"
          />
        </a>
        <div
          className="movieInfoHide"
        >
          <div
            className="bigMovieTitle"
          >
            test2
          </div>
          <div
            className="movieDescription"
          />
          <div
            className="movieGenre"
          >
            Genre: 
            test2 
          </div>
          <div
            className="movieRatings"
            onClick={[Function]}
          >
            <div
              className="imdbRatings"
            >
              <a
                href="https://www.imdb.com/title/tt0132969"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt=""
                  className="ratingImage"
                  src="imdb.png"
                />
              </a>
              (6.9)
            </div>
            <div
              className="userRatings"
            >
              <p
                className="userRatingText"
              >
                User rating:
              </p>
              <div
                className="star-ratings"
                style={
                  Object {
                    "boxSizing": "border-box",
                    "display": "inline-block",
                    "position": "relative",
                  }
                }
                title="0 Stars"
              >
                <svg
                  className="star-grad"
                  style={
                    Object {
                      "height": "0",
                      "position": "absolute",
                      "visibility": "hidden",
                      "width": "0",
                      "zIndex": "0",
                    }
                  }
                >
                  <defs>
                    <linearGradient
                      id="starGrad025328306199083"
                      x1="0%"
                      x2="100%"
                      y1="0%"
                      y2="0%"
                    >
                      <stop
                        className="stop-color-first"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "#eac600",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-first"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "#eac600",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-final"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "rgb(203, 211, 227)",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-final"
                        offset="100%"
                        style={
                          Object {
                            "stopColor": "rgb(203, 211, 227)",
                            "stopOpacity": "1",
                          }
                        }
                      />
                    </linearGradient>
                  </defs>
                </svg>
                <div
                  className="star-container"
                  onClick={null}
                  onMouseEnter={null}
                  onMouseLeave={null}
                  style={
                    Object {
                      "cursor": undefined,
                      "display": "inline-block",
                      "paddingLeft": undefined,
                      "paddingRight": undefined,
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
              </div>
              <p
                className="userRatingText"
              >
                (
                0.00
                ) - 
                0
                 votes
              </p>
            </div>
            <div
              className="giveUserRating"
            >
              <p
                className="userRatingText"
              >
                Give user-rating: 
              </p>
              <div
                className="star-ratings"
                style={
                  Object {
                    "boxSizing": "border-box",
                    "display": "inline-block",
                    "position": "relative",
                  }
                }
                title="0 Stars"
              >
                <svg
                  className="star-grad"
                  style={
                    Object {
                      "height": "0",
                      "position": "absolute",
                      "visibility": "hidden",
                      "width": "0",
                      "zIndex": "0",
                    }
                  }
                >
                  <defs>
                    <linearGradient
                      id="starGrad779329771952828"
                      x1="0%"
                      x2="100%"
                      y1="0%"
                      y2="0%"
                    >
                      <stop
                        className="stop-color-first"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "#eac600",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-first"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "#eac600",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-final"
                        offset="0%"
                        style={
                          Object {
                            "stopColor": "rgb(203, 211, 227)",
                            "stopOpacity": "1",
                          }
                        }
                      />
                      <stop
                        className="stop-color-final"
                        offset="100%"
                        style={
                          Object {
                            "stopColor": "rgb(203, 211, 227)",
                            "stopOpacity": "1",
                          }
                        }
                      />
                    </linearGradient>
                  </defs>
                </svg>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": undefined,
                      "paddingRight": "2px",
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": "2px",
                      "paddingRight": "2px",
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": "2px",
                      "paddingRight": "2px",
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": "2px",
                      "paddingRight": "2px",
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
                <div
                  className="star-container"
                  onClick={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  style={
                    Object {
                      "cursor": "pointer",
                      "display": "inline-block",
                      "paddingLeft": "2px",
                      "paddingRight": undefined,
                      "position": "relative",
                      "verticalAlign": "middle",
                    }
                  }
                >
                  <svg
                    className="widget-svg"
                    style={
                      Object {
                        "height": "20px",
                        "transform": undefined,
                        "transition": "transform .2s ease-in-out",
                        "width": "20px",
                      }
                    }
                    viewBox="0 0 51 48"
                  >
                    <path
                      className="star"
                      d="m25,1 6,17h18l-14,11 5,17-15-10-15,10 5-17-14-11h18z"
                      style={
                        Object {
                          "fill": "rgb(203, 211, 227)",
                          "transition": "fill .2s ease-in-out",
                        }
                      }
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="smallImgStarShow"
        >
          <a
            href="https://www.imdb.com/title/tt0132969"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt=""
              className="ratingImage"
              src="imdb.png"
            />
          </a>
          <div>
            6.9
             / 10
          </div>
        </div>
        <a
          className="movieLink"
          href="#testID2"
        >
          <div
            className="titleTextSmallShow"
          >
            test2
          </div>
        </a>
        <div
          className="rating"
        />
        <img
          alt=""
          className="hideExpandImage"
          src="expanded.png"
        />
      </div>
    </div>
  </div>
</div>
`;
